projects/PointSup/point_sup/detection_utils.py

```diff
@@ -75,16 +74,18 @@ def transform_instance_annotations(
             It will be modified in-place.
         transforms (TransformList or list[Transform]):
         image_size (tuple): the height, width of the transformed image
-        keypoint_hflip_indices (ndarray[int]): see `create_keypoint_hflip_indices`.
+        keypoint_hflip_indices (ndarray[int]): add argument 'keypoint_hflip_indices = None' to prevent unwanted behavior while training with person keys, see `create_keypoint_hflip_indices`. 
     Returns:
         dict:
             the same input dict with fields "bbox", "point_coords", "point_labels"
             transformed according to `transforms`.
             The "bbox_mode" field will be set to XYXY_ABS.
     """
-    annotation = base_transform_instance_annotations(
-        annotation, transforms, image_size, keypoint_hflip_indices
```

Comment:
the correct fix is to use keyword argument passing `keypoint_hflip_indices=keypoint_hflip_indices`.

No other changes are needed.